_source/logzio_collections/_log-sources/aws-control-tower.md

```diff
@@ -0,0 +1,204 @@
+
+
+##### Specify stack details
+
+Specify the stack details as per the table below, check the checkboxes and select **Create stack**.
+
+| Parameter        | Description                                                                                                                                                                                                                                                         | Required/Default   |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| `bucketName`     | Name of the bucket you wish to fetch logs from. Will be used for IAM policy.                                                                                                                                                                                        | **Required**       |
+| `logzioListener` | The Logz.io listener URL for your region. (For more details, see the [regions page](https://docs.logz.io/user-guide/accounts/account-region.html)                                                                                                                   | **Required**       |
+| `logzioToken`    | Your Logz.io log shipping token.                                                                                                                                                                                                                                    | **Required**       |
+| `logLevel`       | Log level for the Lambda function. Can be one of: `debug`, `info`, `warn`, `error`, `fatal`, `panic`.                                                                                                                                                               | Default: `info`    |
+| `logType`        | The log type you'll use with this Lambda. This is shown in your logs under the type field in Kibana. Logz.io applies parsing based on the log type.                                                                                                                 | Default: `s3_hook` |
+| `pathsRegexes`   | Comma-seperated list of regexes that match the paths you'd like to pull logs from.                                                                                                                                                                                  | -                  |
+| `pathToFields`   | Fields from the path to your logs directory that you want to add to the logs. For example, `org-id/aws-type/account-id` will add each of the fields `ord-id`, `aws-type` and `account-id` to the logs that are fetched from the directory that this path refers to. | -                  |
```

Comment:
Can we add here a reference for the "advanced settings" tab, if they're looking for more info?